src/prefect/cli/shell.py

```diff
@@ -0,0 +1,165 @@
+"""
+Provides a set of tools for executing shell commands as Prefect flows.
+Includes functionalities for running shell commands ad-hoc or serving them as Prefect flows,
+with options for logging output, scheduling, and deployment customization.
+"""
+
+import io
+import logging
+import shlex
+from typing import List, Optional
+
+import typer
+from typing_extensions import Annotated
+
+from prefect import flow
+from prefect.cli._types import PrefectTyper
+from prefect.cli.root import app
+from prefect.client.schemas.schedules import CronSchedule
+from prefect.context import tags
+from prefect.deployments.runner import EntrypointType
+from prefect.logging.loggers import get_run_logger
+from prefect.runner import Runner
+from prefect.settings import PREFECT_UI_URL
+from prefect.utilities.processutils import run_process
+
+shell_app = PrefectTyper(name="shell", help="Commands for working with shell commands.")
+app.add_typer(shell_app)
+
+
+async def run_shell_process(
+    command: str,
+    log_output: bool = True,
+):
+    """
+    Asynchronously executes the specified shell command and logs its output.
+
+    This function is designed to be used within Prefect flows to run shell commands as part of task execution.
+    It handles both the execution of the command and the collection of its output for logging purposes.
+
+    Args:
+        command (str): The shell command to execute.
+        log_output (bool, optional): If True, the output of the command (both stdout and stderr) is logged to prefectas.
+                                     Defaults to True.
+
```

Comment:
```suggestion
        log_output (bool, optional): If True, the output of the command (both stdout and stderr) is logged to Prefect.
                                     Defaults to True.

```
noticed this in the param form of the UI

Reply:
wait this gets translated to pydantic parameter descriptions, right? i might be confused where that's happening, in which case this won't actually fix the description